webpack.config.js

```diff
@@ -6,7 +6,7 @@ const LiveReloadPlugin = require('webpack-livereload-plugin');
 
 module.exports = {
   mode: 'development',
-  entry: ['@babel/polyfill', './frontend/src/index.ts'],
```

Comment:
I'm guessing you removed this cos it was deprecated ... it still served a purpose so I'd rather you ensure that purpose is retained by implementing the recommended alternative. https://babeljs.io/docs/en/babel-polyfill.html